### ISSUE TITLE
Lockwood T-Lock Deadbolt - YDD-D4F0 TSDB

### ISSUE THREAD

**Issue by Sian-Lee-SA** [dresden-elektronik/deconz-rest-plugin#5778]
I took a stab at making a DDF but could not get anything for locks. The best I could do was get the battery state. 'm not sure of the current stance on locks within deCONZ but I do wish to have this lock added.

Note*: The name On/Off output 17 was assigned when I tried to map it to a On/Off output in DDF as I was experimenting 

## Device

- Product name: Lockwood T-Lock
- Manufacturer: Yale
- Model identifier: YDD-D4F0 TSDB
- Device type : Please remove all unrelated device types. 
  - Lock

### Identify
![Screenshot 2022-02-10 202220](https://user-images.githubusercontent.com/11685433/153387549-d2b24380-e599-41b0-91db-53b812bb4475.png)
![Screenshot 2022-02-10 202057](https://user-images.githubusercontent.com/11685433/153387564-cb62b394-5054-4505-a901-a1960d80bd3f.png)
![Screenshot 2022-02-10 205324](https://user-images.githubusercontent.com/11685433/153387585-44a05690-eff7-4dcf-99e4-2ae9733ecaa6.png)

### Other clusters that are not mentioned above
![Screenshot 2022-02-10 202358](https://user-images.githubusercontent.com/11685433/153383087-eb2dd0ee-8a4d-40fd-99e1-5cc8f6252fa7.png)

![Screenshot 2022-02-10 202326](https://user-images.githubusercontent.com/11685433/153383066-64fabbf5-e4b5-40d4-81be-8cd68fa085f6.png)
![Screenshot 2022-02-10 202434](https://user-images.githubusercontent.com/11685433/153383154-a93bbda5-d534-431f-81ca-359fc886dbc7.png)

**Comment by Smanar** [comment]
Hello, you are not alone ^^.

Someone else have started one here  https://github.com/dresden-elektronik/deconz-rest-plugin/issues/5778

If you have too much problem can you show your DDF ?

**Comment by Mimiix** [comment]
Closing as it's duplicate

**Comment by Smanar** [comment]
Oups, sorry, mistake.
It's not a duplicate, but someone else is trying DDF for doorlock https://github.com/dresden-elektronik/deconz-rest-plugin/pull/5775

**Comment by Smanar** [comment]
@Sian-Lee-SA you have give up ?

**Comment by Sian-Lee-SA** [comment]
Never really took it on, played around a bit then decided to give Zigbee2MQTT a try which had everything working from get go. I also thought to explore Zigbee2MQTT because every new device I get seems to already be supported. But after using it for a while I have started to see the cracks in it. But their lock for my specific model has everything including the add user and other bits an pieces.

After a few weeks of using Zigbee2MQTT, I had decided to dust of my old Philips Hue bridge specifically for my lights as they weren't playing nice. So right now I have a hybrid setup and personally finding the hybrid setup more stable and compatible atm.
